vispy/visuals/_scalable_textures.py

```diff
@@ -9,6 +9,39 @@
 from vispy.gloo.texture import should_cast_to_f32
 
 
+def get_default_clim_from_dtype(dtype):
+    """Get min and max color limits based on the rsange of the dtype."""
+    # assume floating point data is pre-normalized to 0 and 1
+    if np.issubdtype(dtype, np.floating):
+        return 0, 1
+    # assume integer RGBs fill the whole data space
+    dtype_info = np.iinfo(dtype)
+    dmin = dtype_info.min
+    dmax = dtype_info.max
+    return dmin, dmax
+
+
+def get_default_clim_from_data(data):
+    """ Compute a reasonable clim from the min and max, taking nans into account.
+    If there are no non-finite values (nan, inf, -inf) this is as fast as it can be.
+    Otherwise, this functions is about 3x slower.
+    """
+
+    # Fast
```

Comment:
```suggestion
def get_default_clim_from_data(data):
    """Compute a reasonable clim from the min and max, taking nans into account.
    
    If there are no non-finite values (nan, inf, -inf) this is as fast as it can be.
    Otherwise, this functions is about 3x slower.
    """
    # Fast
```

I believe we have docstring checking explicitly turned off due to all the violations currently in VisPy, but this is how most docstring linters will prefer this to be.